lib/QubitInformationObjectXmlCache.class.php

```diff
@@ -59,25 +59,29 @@ protected function createExportDestinationDirs()
   }
 
   /**
-   * Export information object to EAD (if top-level) and DC.
+   * Export information object to EAD (if top-level) and/or DC.
    *
    * @param object  information object instance to export
+   * @param string  format of XML ('dc', 'ead', or 'all')
    *
    * @return null
    */
-  public function export($resource)
+  public function export($resource, $format = 'all')
```

Comment:
I'd prefer `null` instead of `'all'`, I think it would have been easier to process. Same, not a big deal at this point.